browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1439,6 +1439,9 @@ L.CanvasTileLayer = L.Layer.extend({
 		else if (textMsg.startsWith('cellautofillarea:')) {
 			this._onCellAutoFillAreaMsg(textMsg);
 		}
+		else if (textMsg.startsWith('cellautofillpopuponmouseup:')) {
```

Comment:
more generic name please (no on mouse up in the name)

Reply:
DONE